tools/mock-service-host/README.md

```diff
@@ -42,28 +37,27 @@ Stateless:
 
 -   Mock Service Host will return a valid mocked response for each valid call of GET/List.
 
-# Getting Started (with Azure CLI)
+# Getting Started
 
 ## Prerequisites
 
--   The [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) is well installed.
--   The [Nodejs](https://nodejs.org/en/download/) is well installed.
+- [Nodejs](https://nodejs.org/en/download/) 14.15 or above
 
-## Step 1: Start a Mock Service Host
+## Start Mock Service Host
 
-### 1.1 Clone this project
+### Clone this project
```

Comment:
@changlong-liu  has built the script for one click starting mock-service-host, is it possible to integrate here as well?

Reply:
Yes. I've rewritten this part.

Reply:
Currently quick launching should be available with two commands:
~~~
> npm install mock-service-host
> node node_modules/mock-service-host/dist/src/main.js
~~~

The script I composed in an azure-sdk-for-net PR: https://github.com/Azure/azure-sdk-for-net/pull/26087/files is mainly targeting on "trust self-signed certificate" in windows.  This should be necessary only in .Net Mgmt SDK. I think we can add something like below in mock-service-host doc?
~~~
For .Net mock test, refer to ....
~~~

Reply:
@changlong-liu I think certificate installation for windows is a kind of way to trust self-signed certificate for http call from specific clients. So let us leave the process to the future when we add more client usage part.